### PR TITLE
Remove deprecated themes

### PR DESCRIPTION
**Pull request by illuzan**
-Removed themes that are not available on Playstore anymore.

### CODE REVIEW

File: substratum_tab_dark_themes.xml [hunk 1]
```diff
@@ -54,12 +42,6 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/compound.jpg</backgroundimage>
     </theme>
-    <theme id="Cosmos">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: let this be

File: substratum_tab_dark_themes.xml [hunk 2]
```diff
@@ -102,12 +84,6 @@
         <pricing>Free</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/derty_dark.jpg</backgroundimage>
     </theme>
-    <theme id="DNA Dark Project">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: ditto

File: substratum_tab_dark_themes.xml [hunk 3]
```diff
@@ -174,24 +138,6 @@
         <pricing>Free</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/m5theme.png</backgroundimage>
     </theme>
-    <theme id="Magpie">
-        <author>Jimmy Setiawan</author>
-        <package>com.setiawanjimmy.TheMagpie</package>
-        <pricing>Paid</pricing>
-        <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/magpie.png</backgroundimage>
-    </theme>
-    <theme id="ModernUI Free">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: ditto

File: substratum_tab_dark_themes.xml [hunk 4]
```diff
@@ -174,24 +138,6 @@
...
-        <author>Laszlo Szecsenyi</author>
-        <package>sub.kickoff.modernui</package>
-        <pricing>Free</pricing>
-        <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/modernuifree.png</backgroundimage>
-    </theme>
-    <theme id="ModernUI Pro">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: ditto

File: substratum_tab_dark_themes.xml [hunk 5]
```diff
@@ -210,12 +156,6 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/nineteen.jpg</backgroundimage>
     </theme>
-    <theme id="Onyx">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: ditto

File: substratum_tab_dark_themes.xml [hunk 6]
```diff
@@ -344,16 +278,4 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/victory.png</backgroundimage>
     </theme>
-    <theme id="View">
```
Comment (MSF-Jarvis) [inline substratum_tab_dark_themes.xml]: ditto

File: substratum_tab_light_themes.xml [hunk 7]
```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <substratum>
-    <theme id="Aurora">
```
Comment (MSF-Jarvis) [inline substratum_tab_light_themes.xml]: ditto

File: substratum_tab_light_themes.xml [hunk 8]
```diff
@@ -24,24 +18,12 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/belo.png</backgroundimage>
     </theme>
-    <theme id="Black Light">
```
Comment (MSF-Jarvis) [inline substratum_tab_light_themes.xml]: ditto

File: substratum_tab_light_themes.xml [hunk 9]
```diff
@@ -72,12 +54,6 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/flare.jpg</backgroundimage>
     </theme>
-    <theme id="Flatty Light">
```
Comment (MSF-Jarvis) [inline substratum_tab_light_themes.xml]: ditto

File: substratum_tab_light_themes.xml [hunk 10]
```diff
@@ -174,12 +138,6 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/harmony.jpg</backgroundimage>
     </theme>
-    <theme id="NovaUI">
```
Comment (MSF-Jarvis) [inline substratum_tab_light_themes.xml]: ditto

**Review by msfjarvis — CHANGES_REQUESTED** [review]

File: substratum_tab_clear_themes.xml [hunk 11]
```diff
@@ -54,12 +48,6 @@
         <pricing>Paid</pricing>
         <backgroundimage>https://raw.githubusercontent.com/substratum/database/master/images/Sprite.jpg</backgroundimage>
     </theme>
-    <theme id="ThaJoker SupernovaRemix">
```
[the remark this answers is not in the document]
Comment (allstargaurav) [reply substratum_tab_clear_themes.xml]: Ok, But its removed from play store

[the remark this answers is not in the document]
Comment (MSF-Jarvis) [reply substratum_tab_clear_themes.xml]: > Ok, But its removed from play store

Oh, nuke it then.

File: substratum_tab_light_themes.xml (hunk 7 above)
Comment (allstargaurav) [reply substratum_tab_light_themes.xml]: Its not removed from playstore but its old af, there are more themes by him
what should i do?

Comment (MSF-Jarvis) [reply substratum_tab_light_themes.xml]: I'll email the themer and see what he has to say.

**Comment by MihailMihov** [comment]
Christmas theme just got an update
https://play.google.com/store/apps/details?id=me.piere.christmas

**Pull request closed without merging**